docs/csharp/getting-started/with-visual-studio.md

```diff
@@ -12,78 +12,74 @@ ms.devlang: csharp
+1. Launch Visual Studio 2017. Select **File > New > Project** from the menu bar. In the **Add New Project** dialog, select the **.NET Core** node followed by the **Console App (.NET Core)** project template. In the **Name** text box, type "HelloWorld". Select the **OK** button.
 
-2. In the right-hand pane, choose **Console App (.NET Core)**. Enter the name of the project, `HelloWorld`, and make sure that the **Create directory for solution** box is checked, as the following figure shows.
-
-   ![Screenshot showing the New Project dialog with Console App selected](./media/with-visual-studio/vs_newproject.jpg)
+   ![New Project dialog with Console App selected](./media/with-visual-studio/newproject.png)
    
-3. Choose the **OK** button. Visual Studio displays its development environment with its code window, as the following figure shows. The C# Console Application template for .NET Core automatically defines a class, `Program`, with a single method, `Main`, that takes a @System.String array as an argument. `Main` is the application entry point, the method that is called automatically by the runtime when it launches the application. Any command-line arguments supplied when the application is launched are available in the *args* array.
+1. Visual Studio loads the development environment. The C# Console Application template for .NET Core automatically defines a class, `Program`, with a single method, `Main`, that takes a @System.String array as an argument. `Main` is the application entry point, the method that's called automatically by the runtime when it launches the application. Any command-line arguments supplied when the application is launched are available in the *args* array.
 
-   ![Visual Studio and the new HelloWorld project](./media/with-visual-studio/vs_devenv.jpg)
+   ![Visual Studio and the new HelloWorld project](./media/with-visual-studio/devenv.png)
 
-   The template creates a very simple "Hello world" application -- it calls the @System.Console.WriteLine(System.String) method to display the literal string "Hello World!" to the console window. By selecting the "HelloWorld" button with the green arrow on the toolbar, you can run the program in Debug mode now. If you do, though, the console window is visible for only a very brief time interval before it closes. This occurs because the `Main` terminates and the application ends as soon as the single statement in the `Main` method executes.
+   The template creates a simple "Hello World" application. It calls the @System.Console.WriteLine(System.String) method to display the literal string "Hello World!" in the console window. By selecting the "HelloWorld" button with the green arrow on the toolbar, you can run the program in Debug mode. If you do, the console window is visible for only a brief time interval before it closes. This occurs because the `Main` terminates and the application ends as soon as the single statement in the `Main` method executes.
```

Comment:
Rather than using the @ syntax, it's best to use <xref> with ?displayProperty=fullName here.
the Main --> the Main method